### ISSUE TITLE
List - `onOrder` buttons should be selectable by hitting the 'Space' key

### ISSUE THREAD

**Issue by jcfilben** [grommet/grommet#7131]
### Expected Behavior
The onOrder buttons in the list should respond to pressing 'Enter' or 'Space'.

### Actual Behavior
Pressing 'Enter' works but 'Space' does not work.

### URL, screen shot, or Codepen exhibiting the issue
https://storybook.grommet.io/?path=/story/visualizations-list-order--order&globals=theme:hpe

### Steps to Reproduce

1. Go to the above story
2. Navigate to a up/down button on the list and hit 'Space' on the keyboard.

### Your Environment
- Grommet version:
- Browser Name and version:
- Operating System and version (desktop or mobile):
